File: cpp/bc1177.cpp
#include <iostream>

using namespace std;

int main()
{
	int vec[1000], val, j = 0;
	
	cin >> val;
	
	for(int i = 0; i < 1000; i++)
	{
		vec[i] = j;
		if(j == val-1) j = 0;
		else j++;
	}
	
	for(int i = 0; i < 1000; i++)
		cout << "N[" << i << "] = " << vec[i] << endl;
}

File: python/bc1177.py
vec = [None]*1000
i = 0
val = int(input())

for c in range(0, 1000):
    vec[c] = i

    if i == val-1: i = 0
    else: i += 1

for j in range(0, 1000):
    print(f"N[{j}] = {vec[j]}")